src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
```

Comment:
Consider using try-with-resources for the `Session` to ensure it is always closed, even if an exception occurs. However, since you are managing the transaction manually, ensure that the transaction is committed or rolled back before the session is closed.